### ISSUE TITLE
Create a way to use symbols in a term (ex. pi)

### ISSUE THREAD

**Issue by njericha** [JacquesCarette/literate-scientific-software#167]
We would like to have a way to capture the concept of pi but there is no way to have the term for pi be the symbol π. This would also be used so we can define degrees as a derived unit in terms of pi.

**Comment by smiths** [comment]
Could address by changing string type to text type (discussion during Nov 2, 2017 meeting).

**Comment by JacquesCarette** [comment]
Actually, as we've subsequently discovered, Haskell's `String` type is unicode, so this can be done right now. No extra code to write.  And the backends (LaTeX and HTML) have been fixed to deal with UTF-8 properly, so we're good to go.

So the hack in GamePhysics can be removed! @elwazana would you like to try to deal with it?

**Comment by JacquesCarette** [comment]
Yes. `pi` is defined on line 272 of `Unitals.hs`.  The part that uses `Greek Pi_L` will be fixed by @deviprasad135 . The term part (the `nounPhaseSP`) is now "pi" but it should be "π" instead.

**Comment by elwazana** [comment]
Should I also change the <code>mkQuant</code> from "pi" to "π"?

https://github.com/JacquesCarette/Drasil/blob/cad5e614b0cfa68ce05f69ef7944bdfa53df77a6/code/Example/Drasil/GamePhysics/Unitals.hs#L273

**Comment by JacquesCarette** [comment]
No, as that's the internal id.  Even though it could be unicode too, I want to retain that part in ASCII for debugging purposes.

**Comment by niazim3** [comment]
Can this issue be closed with the merge of [this pull request](https://github.com/JacquesCarette/Drasil/pull/509)?

**Comment by niazim3** [comment]
@JacquesCarette How is "\Data\Drasil\Quantities\Math.hs" as a location?

**Comment by niazim3** [comment]
@JacquesCarette Should stable also be updated before this issue is closed to remove the lines from Chipmunk_log.log? If so, can that be done in the same pull request?

**Comment by szymczdm** [comment]
@niazim3 `Data.Drasil.Quantities.Math` is fine for now. I think we may end up moving it to a different namespace later on if we restructure Data.Drasil for clarity.

Also, if that's the only change (as per my comment on the PR), then yes. That needs to be fixed and then once the travis build comes back passing, we can merge the PR in and close this issue.
